CHANGES.rst

```diff
@@ -17,9 +17,13 @@ astropy.coordinates
 ^^^^^^^^^^^^^^^^^^^
 
 - The ``Distance`` object now accepts ``parallax`` as a keyword in the
-  initializer, and supports retrieving a parallax (as an ``Angle``) via 
+  initializer, and supports retrieving a parallax (as an ``Angle``) via
   the ``.parallax`` attributes. [#6855]
 
+- Coordinate frame classes now can't be added to the frame transform graph if
```

Comment:
Technically this is an API change, right? Should probably put it there instead of "new feature"